app/main.py

```diff
@@ -1,8 +1,47 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self,
+                 comfort_class: int,
+                 clean_mark: int,
+                 brand: str
+    ) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(
+            self,
+            distance_from_city_center: float,
+            clean_power: int,
+            average_rating: float,
+            count_of_ratings: int
+    ) -> None:
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def serve_cars(self, cars: list) -> float:
+        if len(cars) == 0:
+            return 0
+        income = 0
+        for car in cars:
+            if self.clean_power > car.clean_mark:
+                income += self.calculate_washing_price(car)
+                self.wash_single_car(car)
+            return round(income, 1)
+
+    def calculate_washing_price(self, car: "Car") -> float:
+        income = (car.comfort_class * (self.clean_power - car.clean_mark) *
+                  (self.average_rating/self.distance_from_city_center))
+        return income
+
+    def wash_single_car(self, car: "Car") -> None:
```

Comment:
Do not use string annotation, you could pass `Car` here and in other methods as well directly without `""`.

Reply:
I had had problems with annotation in previous tasks so did as earlier.
fixed.